src/software/test_util/equal_within_tolerance.h

```diff
@@ -48,6 +49,21 @@ namespace TestUtil
                                                     const Polygon &poly2,
                                                     double tolerance);
 
+    /**
+     * Checks if two stadiums are within tolerance of each other
+     * Two stadiums are within tolerance of each other if the corresponding points are
+     * within tolerance of each other and the radii are within tolerance
+     *
+     * @param stadium1, stadium2 Polygons to compare
+     * @param tolerance tolerance to check equality with, default is
+     * METERS_PER_MILLIMETER
+     *
+     * @return AssertionSuccess if the two polygons are within tolerance of each other
```

Comment:
nit:
```suggestion
     * @param stadium1, stadium 2 stadiums to compare
     * @param tolerance tolerance to check equality with, default is
     * METERS_PER_MILLIMETER
     *
     * @return AssertionSuccess if the two stadiums are within tolerance of each other
```
Shouldn't the default value of `tolerance` also be `METERS_PER_MILLIMETER`? (I think it would then need to be the same for the function above and below